war/src/main/less/modules/side-panel-tasks.less

```diff
@@ -44,12 +44,12 @@
 }
 .task-link:link,
 .task-link:visited {
-  color: @text-color;
+  color: var(--text-color);
   text-decoration: none;
 
   &:hover,
   &:focus {
-    background-color: #F8F8F8;
+    background-color: var(--task-link-background-color--hover);
```

Comment:
I'm thinking that maybe we should just treat these links and the ones in the footer as a dark link variant. They all behave the same, and I don't think we should be hyper specific here.

Reply:
they are used for different parts of the interface though, one is used for background colour and this is used for hover, bit wary of overloading variables, internally they are re-using the same colour variable, but they are different parts of the interface...

Reply:
I didn't see this was about bg color nevermind. My comments were about color and font-weight basically.